src/components/form/CharSheetData.js

```diff
@@ -121,8 +121,8 @@ const charsheet = {
         // the number can modify some values, but is likely context specific and would best be left to the user
         // there doesn't seem to be a specific minimum or maximum number of backgrounds a character can have (most of Tim's have 2 or 3), so we may need to allow the user to add more fields
         // 3 sets fields by default may be good, if the user doesn't input values print blank?
-        background_numbers: [],
-        background_names: [],
+        background_numbers: [""],
```

Comment:
Small concern with initializing these to the empty string is they will count as valid backgrounds to add to the pdf, which may result in some unwanted behavior if these are not modified. I think we'll be good here since we're just printing the contents of these arrays and not adding anything else to the string, but could be a concern in other areas such as icon relationships, where I add a colon between values. If there is an empty string, this will print blank space with a colon in between. We could implement null checks here or leave these in and implement empty string checks when printing to the pdf. Food for thought!